Debonair-Squids/Bangazon_WebApp
# Display Invoice (Empty)

# Steps To Test
1. git fetch --all
2. git checkout TB-Working
3. run the app
4. sign into an existing user.
5. click on "View Cart" from the nav
6. you should see a message saying "You do not currently have an open invoice"


### Description of what should happen when steps are followed
you should see a message saying "You do not currently have an open invoice"

**At least two people needs to approve pull request before merged to master**
